src/main/java/org/folio/verticles/DataImportConsumerVerticle.java

```diff
@@ -0,0 +1,77 @@
+package org.folio.verticles;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Promise;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.kafka.AsyncRecordHandler;
+import org.folio.kafka.GlobalLoadSensor;
+import org.folio.kafka.KafkaConfig;
+import org.folio.kafka.KafkaConsumerWrapper;
+import org.folio.kafka.KafkaTopicNameHelper;
+import org.folio.kafka.SubscriptionDefinition;
+import org.folio.processing.events.EventManager;
+import org.folio.util.pubsub.PubSubClientUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.support.AbstractApplicationContext;
+import org.springframework.stereotype.Component;
+
+import static org.folio.DataImportEventTypes.DI_EDIFACT_RECORD_CREATED;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
```

Comment:
Could you please clarify : Why SCOPE_PROTOTAPE is needed here?

Reply:
sure, there is an ability to deploy multiple instances of the `DataImportConsumerVerticle` verticle and a new verticle instance should be created per every verticle deployment